src/ngx_http_lua_headers.c

```diff
@@ -1322,9 +1322,13 @@ ngx_http_lua_ffi_set_resp_header(ngx_http_request_t *r, const u_char *key_data,
 
 
 int
-ngx_http_lua_ffi_req_header_set_single_value(ngx_http_request_t *r,
-    const u_char *key, size_t key_len, const u_char *value, size_t value_len)
+ngx_http_lua_ffi_req_set_header(ngx_http_request_t *r,
```

Comment:
We should align this function's name with `ngx_http_lua_ffi_set_resp_header`. It should thus be `ngx_http_lua_ffi_set_req_header`.